pkg/constants/constants.go

```diff
@@ -183,4 +183,28 @@ const (
 	// has completed the sync of the volume. This annotation is set by the CSI driver
 	// when the sync is completed.
 	CSIVSphereVolumeSyncAnnotationValueCompleted = "completed"
+
+	// AsyncQuotaPerformCheckAnnotationKey is the annotation key indicating that the ContentLibrary
+	// associated with a VirtualMachinePublishRequest is subject to VCFA quota approval.
+	AsyncQuotaPerformCheckAnnotationKey = "vmware-system-vcfa-storage-quota"
+
+	// AsyncQuotaCheckRequestedCapacityAnnotationKey is the annotation key added
+	// by the VirtualMachinePublishRequest mutation webhook to initiate quota validation. Adding
+	// this annotation will indicate that the controller must provide the requested capacity
+	// and block request fulfillment until further notice.
+	AsyncQuotaCheckRequestedCapacityAnnotationKey = "vcfa.storage.quota.check.vmware.com/requested-capacity"
```

Comment:
should we add a validating webhook to ensure it is not writable by unprivileged users?

Reply:
this needed to be updated. the original intent was to have a mutation webhook handle adding this annotation. this has since been changed to remove the mutation webhook from the flow and have this handled solely by the controller. will update the comment to be accurate. good catch here